Add hooks to make.config for inserting our CC, CXX, flags.

diff -Naurb symtabAPI.orig/src/dyninst/make.config symtabAPI/src/dyninst/make.config
--- symtabAPI.orig/src/dyninst/make.config	2009-12-08 12:03:19.000000000 -0600
+++ symtabAPI/src/dyninst/make.config	2011-02-02 00:25:00.000000000 -0600
@@ -100,12 +100,18 @@
 # But if the file doesn't exist there (i.e. if the user hasn't checked them out
 # with cvs), then we should look in a backup location.  That's what this vrble
 # is about.  It's used several times below in this file.
-GCC      ?= gcc
-GXX      ?= g++
+
+HPC_CC =
+HPC_CXX =
+HPC_CFLAGS =
+HPC_CXXFLAGS =
+
+GCC      = $(HPC_CC)
+GXX      = $(HPC_CXX)
 LINKER   = ld
 AS      	= as
-CC       ?= $(GCC)
-CXX      ?= $(GXX)
+CC       = $(HPC_CC)
+CXX      = $(HPC_CXX)
 LEX      = flex
 RANLIB   = ranlib
 AR       = ar
@@ -466,8 +472,8 @@
 endif
 
 # flags for normal (non-kludge) files, with common warnings
-CFLAGS		= $(IFLAGS) $(UNIFIED_DEF) $(OPT_FLAG) $(COMMON_WARNINGS) $(ARCH_FLAGS)
-CXXFLAGS	= $(IFLAGS) $(UNIFIED_DEF) $(OPT_FLAG) $(COMMON_WARNINGS) $(STDINT_MACROS) $(ARCH_FLAGS)
+CFLAGS   = $(HPC_CFLAGS) $(IFLAGS) $(UNIFIED_DEF) $(OPT_FLAG) $(COMMON_WARNINGS) $(ARCH_FLAGS)
+CXXFLAGS = $(HPC_CXXFLAGS) $(IFLAGS) $(UNIFIED_DEF) $(OPT_FLAG) $(COMMON_WARNINGS) $(STDINT_MACROS) $(ARCH_FLAGS)
 LDFLAGS		= $(ARCH_FLAGS)
 
 # flags for kludge files, won't have warnings generated
